src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -166,15 +170,15 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
               ? ""
               : html`
                   <paper-icon-button
-                    .icon=${"hass:skip-next"}
+                    icon=${"hass:skip-next"}
                     .disabled="${OFF_STATES.includes(stateObj.state)}"
                     .action=${"media_next_track"}
                     @click=${this._handleClick}
                   ></paper-icon-button>
                 `}
           </div>
           <paper-icon-button
-            .icon=${"hass:dots-vertical"}
+            icon=${"hass:dots-vertical"}
             @click="${this._handleMoreInfo}"
```

Comment:
Also we should drop the `""` on dynamic values.
```suggestion
            @click=${this._handleMoreInfo}
```